This patch fixes API updates in Java 7.
The package can be built with Java 7 with this patch.
--- a/src/java/com/jamonapi/proxy/JAMonDataSource.java
+++ b/src/java/com/jamonapi/proxy/JAMonDataSource.java
@@ -77,5 +77,9 @@
     public Reference getReference() throws NamingException {
         return ((Referenceable)realDataSource).getReference();
     }
+
+    public java.util.logging.Logger getParentLogger() throws java.sql.SQLFeatureNotSupportedException {
+        throw new UnsupportedOperationException("getParentLogger");
+    }
 
 }
--- a/src/java/com/jamonapi/proxy/JAMonDriver.java
+++ b/src/java/com/jamonapi/proxy/JAMonDriver.java
@@ -245,6 +245,10 @@
             
     }
     
+    public java.util.logging.Logger getParentLogger() throws java.sql.SQLFeatureNotSupportedException {
+        throw new UnsupportedOperationException("getParentLogger");
+    }
+
   
 }
 
